app/src/main/java/org/dharmaseed/android/PlayTalkActivity.java

```diff
@@ -60,23 +62,44 @@ public class PlayTalkActivity extends AppCompatActivity
     int talkID;
     DBManager dbManager;
     boolean userDraggingSeekBar;
-    int userSeekBarPosition;
+
+    Timer timer;
 
     private static Talk talk;
 
     private static final String LOG_TAG = "PlayTalkActivity";
 
+    protected static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+
     // request code for writing external storage (the number is arbitrary)
     private static final int PERMISSIONS_WRITE_EXTERNAL_STORAGE = 9087;
 
+    protected void prepareTalkPlayerFragment(TalkPlayerFragment talkPlayerFragment)
```

Comment:
I've just updated the signature of the prepareTalkPlayerFragment function: I don't think we need an argument at all, since we're always interested in this.talkPlayerFragment anyhow! I have included a non-null check of this.talkPlayerFragment, though.

Reply:
Oh yeah, good catch! :)